src/ShopBundle/Bridge/Chameleon/Migration/Script/update-1549299700.inc.php

```diff
@@ -0,0 +1,46 @@
+<h1>Build #1549299700</h1>
```

Comment:
I would probably give these updates totally different numbers.

(1549299689 +1 would be 1549299690 :-) )

Reply:
After a test. I changed the numbers to one before the core updates, so they will run first and require the core updates.